autopilot/contractor.go

```diff
@@ -659,11 +740,35 @@ func (c *contractor) candidateHosts(cfg Config, wanted int) ([]consensus.PublicK
 	return selected, nil
 }
 
-func (c *contractor) isStopped() bool {
-	select {
-	case <-c.ap.stopChan:
-		return true
-	default:
+func calculateHostCollateral(cfg Config, settings rhpv2.HostSettings, renterFunds, txnFee types.Currency) (types.Currency, error) {
```

Comment:
Might be good to also decide on whether we want this in the contractor or worker for formations/renewals. Right now it's split but we should probably have that be consistent.

Reply:
The way we handle collateral is quite confusing. On contract formation we calculate it in the contractor, which I think makes sense because it's based on the expected storage. On renewals however it seems like the collateral is based purely on the existing file contract, so I'm not sure if we even need to take collateral into account on renewals at all?